apps/search-poc/src/config.ts

```diff
@@ -96,6 +96,19 @@ export function getCollectionEModelsQuery(resourceId) {
     } LIMIT 100`;
 }
 
+export function getCollectionReconstructedCellsQuery(resourceId) {
+  return `
+  prefix nxv: <https://neuroshapes.org/>
+  prefix schema: <http://schema.org/>
+  prefix nexus: <https://bluebrain.github.io/nexus/vocabulary/>
+  SELECT * WHERE {
+    <${resourceId}> nxv:reconstructedcells ?reconstructedcell .
+    ?reconstructedcell schema:name ?name .
+    ?reconstructedcell nexus:self ?self .
+    optional {?reconstructedcell schema:description  ?description}
+  } LIMIT 2000`;
```

Comment:
I think we need to introduce pagination. You can pass the offset as a param from your function and add:
```suggestion
  } LIMIT 20 OFFSET ${offset}`;
```

Reply:
Limit has been set to 20, without pagination for now. Going to implement that in separate PR